bodhi/server/models.py

```diff
@@ -1259,6 +1272,41 @@ def signed(self):
             return True
         return all([build.signed for build in self.builds])
 
+    @property
+    def ci_status(self):
+        """ Returns a boolean representing if all the builds associated with
+        this update are ignored or have passed their CI testing.
+
+        Returns False if at least one of the tests of the builds associated
+        with this update are queued, running or failed.
+        """
+
+        ci_status = set([build.ci_status for build in self.builds])
+        if not ci_status.difference(set([CiStatus.ignored, CiStatus.passed])):
+            return True
+        return False
+
+    @property
+    def ci_status_label(self):
```

Comment:
I recommend renaming this to ```ci_status``` and having it just return the Enum. Then you can use the Enum in the template and the human readable bit should "just work".